demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -28,6 +28,7 @@ class IntegrationScript(ContentItem):
     docker_image: Optional[str]
     description: Optional[str]
     is_unified: bool = Field(False, exclude=True)
+    python_version: Optional[str]
```

Comment:
Perhaps `Optional[Version]`?

Reply:
Unfortunately we have to use only primitive values as fields for now